runelite-client/src/main/java/net/runelite/client/plugins/screenshot/ScreenshotPlugin.java

```diff
@@ -346,6 +347,19 @@ public void onChatMessage(ChatMessage event)
 			}
 		}
 
+		if (config.screenshotGauntletLoot() && chatMessage.equals(GAUNTLET_LOOTED_MESSAGE))
+		{
+			try
```

Comment:
Nightfire was saying to set a flag here (i.e. `gauntletScreenshot`) and then use the `ItemContainerChanged` event to take the screenshot.

For Example:
```java
	@Subscribe
	public void onItemContainerChanged(ItemContainerChanged c)
	{
		if (!gauntletScreenshot || c.getContainerId() != InventoryID.INVENTORY.getId())
		{
			return;
		}
		
		gauntletScreenshot = false;
		takeScreenshot("Gauntlet loot", "Gauntlet loot");
	}
```

This is how the loot tracker, and other plugins, handles similar situations.

Reply:
Awesome. I actually have something similar on my local, but time got away from me

When I'm off from work tonight I'll apply this to what I had brewing